File: src/Core/Houston.Application/CommandHandlers/ConnectorFunctionHistoryCommandHandlers/Update/UpdateConnectorFunctionHistoryCommand.cs
namespace Houston.Application.CommandHandlers.ConnectorFunctionHistoryCommandHandlers.Update {
	public sealed record UpdateConnectorFunctionHistoryCommand(Guid Id, byte[] Script, byte[] Package, List<UpdateConnectorFunctionInputCommand> Inputs) : IRequest<IResultCommand>;

	public sealed record UpdateConnectorFunctionInputCommand(Guid? Id, InputType InputType, string Name, string Placeholder, string Replace, bool Required, string? DefaultValue, string[]? Values, bool AdvancedOption);
}

File: src/Web/WebAngular/src/app/domain/commands/connector-function-history-commands/update-connector-function-history.command.ts
import { UpdateConnectorFunctionInputCommand } from "./update-connector-function-input.command";

export interface UpdateConnectorFunctionHistoryCommand {
	id: string;
	script: Buffer | Uint8Array | string;
	package: Buffer | Uint8Array | string;
	inputs: UpdateConnectorFunctionInputCommand[];
}
